moto/sns/models.py

```diff
@@ -284,9 +285,9 @@ def set_endpoint_attributes(self, arn, attributes):
             }
         }
     }]
-}
+})
```

Comment:
Even in the JSON representation, the *_policy are represented as string (not as nested structure)